src/schema/v2/me/showsConnection.ts

```diff
@@ -30,17 +38,36 @@ export const ShowsConnection: GraphQLFieldConfig<void, ResolverContext> = {
   }),
   description:
     "A list of shows for the user (pagination logic might be broken)",
-  resolve: async (_root, args, { meShowsLoader, requestLocationLoader }) => {
+  resolve: async (
+    _root,
+    args,
+    { ipAddress, meShowsLoader, requestLocationLoader }
+  ) => {
     if (!meShowsLoader) return null
 
     const { limit: size, offset } = getPagingParameters(args)
-    const { near, ip, page, sort, status } = args
+    const {
+      near,
+      includeShowsNearIpBasedLocation,
+      ip,
+      page,
+      sort,
+      status,
+    } = args
 
     if (ip && near) {
```

Comment:
does the new argument need to be added to this list?

Reply:
Good idea!